plug-ins/fight/onehit_undef.cpp

```diff
@@ -1274,8 +1281,12 @@ void UndefinedOneHit::damApplyMasterHand()
     dam_bonus += (skill_level(*gsn_mastering_pound, ch) - victim->getModifyLevel()) * level_mod * 100;
     dam_bonus += skill_level(*gsn_mastering_pound, ch) / 10;
     dam_bonus = (int)URANGE(1, (int)dam_bonus, 20);
+    d.log(dam, "dam");
+    d.log(dam_bonus, "dam_bonus");
+    d.log(3 + ch->getModifyLevel() / 10, "old_dam_bonus");
```

Comment:
Current debug output is very hard to read — especially in real time for every single hit in combat. Suggestion: use more human-legible format: 
- “—— damage bonus: {W60{x ( {c13 d 10{x )

Reply:
@fiorine 
Here's current output:
```
mpound chances: skill={C20{x, stats={C12{x, lvl={C13{x, quick={C8{x, final={C8{x
```
How do you want the whole string to change? Is one line ok or line breaks would help?